### ISSUE TITLE
Inquiry about Unimplemented Functions in Key2Mesh

### ISSUE THREAD

**Issue by Aom-sushar** [swampxx/Key2Mesh#1]
Hello,
Thank you for open-sourcing the code for your paper! I noticed that there are two functions in the key2mesh.py file that are not implemented:
<img width="582" alt="截屏2024-10-09 下午10 49 17" src="https://github.com/user-attachments/assets/cc2faf5f-a974-420e-aab1-68b56d8a863a">

Could you please advise on how these two functions should be implemented? Would it be possible to provide the complete code for training the model?

Thank you again!

**Comment by swampxx** [comment]
Thank you for your interest in our paper! 

Unfortunately, we won’t be releasing the training part of the code due to some restrictions. However, I recommend checking out the following resources, which should be helpful in implementing the training process:
* [wdgrl.py](https://github.com/jvanvugt/pytorch-domain-adaptation/blob/master/wdgrl.py)
* [Learned Gradient Descent](https://github.com/InpatientJam/Learned-Gradient-Descent)
